cairo/tests/utils/strategies.py

```diff
@@ -362,7 +361,7 @@ def tuple_strategy(thing):
             secured=st.just(True),
             default=st.none(),
             _data=st.fixed_dictionaries(
-                {address: st.from_type(Account) for address in addresses}
+                {address: account_strategy for address in addresses}
```

Comment:
modify a `st.register_type_strategy(Account, st.builds(Account))` with account_strategy and here we can keep st.from_type(Account)